apps/ledger-live-desktop/tests/specs/speculos/add.account.spec.ts

```diff
@@ -22,7 +23,7 @@ for (const currency of currencies) {
   test.describe("Add Accounts", () => {
     test.use({
       userdata: "skip-onboarding",
-      speculosCurrency: currency,
+      speculosApp: specs[currency.deviceLabel.replace(/ /g, "_")],
```

Comment:
why don't you do replace directly in fixture instead of doing it in every test file?

Reply:
wouldn't work for Exchange, plugins